docs/topics/developer-lightspeed/proc_running-rag-analysis.adoc

```diff
@@ -0,0 +1,44 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="running-rag-analysis_{context}"]
+= Running an analysis by using solution server
+
+[role="_abstract"]
+Solution server uses Retrieval Augmented Generation (RAG) to generate a resolution based on a context. Solution server derives the context from rules and past changes to similar issues in the codebase of your organization and generates a prompt for your large language model (LLM). The LLM generates migration hints for the issues that it detects based on the contextual prompt. 
+
+An analysis completed by the solution server identifies the issues in the code. It displays the success metrics for incidents and issues. Success metrics indicate the confidence level in applying the code fix based on how many times the update was applied in past analysis.
+
+.Prerequisites
+
+* You installed the {ProductShortName} distribution version 8.0.0 in your system.
+* You installed Jave 17+ and Maven 3.9.9+ in your system. 
+* You installed the {ProductFullName} extension version 8.0.0 in VS Code. 
+* You installed the latest version of Language Support for Java(TM) by Red Hat extension in Visual Studio (VS) Code.
+* You opened a Java project in your VS Code workspace.
+//check what's the alternative for Konveyor references in the d/s build.
+* You configured an analysis profile on the *Konveyor Analysis View* page.
+
+.Procedure
+
+. Verify that solution server is enabled in one of the following ways:
++
+.. Type `Ctrl + Shift + P` in VS Code search (Linux/Windows system) and `Cmd + Shift + P` for Mac to go to the command palette. 
+.. Enter `Preferences: Open User Settings (JSON)` to open the `settings.json` file.
+.. Ensure that `konveyor.solutionServer.enabled` is set to `true`.
++
+OR
++
+.. Go to *Extensions > {mta-dl-plugin} > settings*
+.. Click the *Solution Server:Enabled* option to enable the server. 
++
+. Click the {mta-dl-plugin} extension and click *Open Konveyor Analysis View*.
++
+. Select a profile for the analysis.
++
+. Click *Start* to start the {ProductShortName} RPC server.
++
+. Click *Run Analysis* on the  *Konveyor Analysis View* page.
+
+To resolve the identified issues, see Applying resolutions after a solution server analysis.
```

Comment:
> after a solution server analysis.

Note: Solution server is not running an analysis, assume we will want to reword this.